index.html

```diff
@@ -30,19 +30,21 @@ <h1 class='f3 fw2 montserrat aqua ttu'>CID Inspector</h1>
   <section class='pv4 pl2 pl5-l'>
     <div class='pt2 measure-wide cf pr2 pr0-l'>
       <a class='gray hover-aqua link underline fr ml3' href="https://proto.school/#/anatomy-of-a-cid">Tutorial</a>
-      <a class='gray hover-aqua link underline fr' href="https://github.com/ipld/cid">Spec</a>
+      <a class='gray hover-aqua link underline fr ml3' href="https://github.com/ipld/cid">Spec</a>
+      <a class='gray hover-aqua link underline fr' href="https://docs.ipfs.io/concepts/content-addressing/#identifier-formats">Docs</a>
       <label for="input-cid" class='db pb2 w-100 tracked ttu f5 teal'>CID</label>
       <input id="input-cid" type="text" class='db w-100 ph1 pv2 monospace input-reset ba b--black-20 border-box' />
```

Comment:
The worst thing about this awesome page is that it lacks all context. (It has great SEO so it can happen, but it's only in the search engine that it says what it's for.) Could we add a placeholder to the input field as a clue to what the heck you're supposed to do once you've landed here?"  eg... 

Paste a CID here to break it down! 
Paste an IPFS CID here to break it down! (only if it can't be used for Filecoin, other projects)
Paste a CID here  for a human-readable breakdown!

![image](https://user-images.githubusercontent.com/19171465/96290168-330dbb80-0fb4-11eb-8436-eb223ab0d3c1.png)
![image](https://user-images.githubusercontent.com/19171465/96290366-77995700-0fb4-11eb-9ea4-d9b77b33e0f3.png)



```suggestion
      <input id="input-cid" type="text" placeholder="Paste a CID here to break it down!" class='db w-100 ph1 pv2 monospace input-reset ba b--black-20 border-box' />
```